Add-ons/Umbraco-Plumber/Licensing/index.md

```diff
@@ -0,0 +1,21 @@
+---
+meta.Title: "Umbraco Plumber Licensing"
+meta.Description: "Here you can find information about how to use a license with Umbraco Plumber"
+versionFrom: 8.0.0
+versionTo: 10.0.0
+---
+
+# Licensing
+
+Umbraco Plumber is a licensed product but doesn't require a purchase to use. New installs are defaulted to a trial license, while the paid license is available for purchase. The trial license introduces some restrictions around advanced features but is otherwise a full-featured workflow platform. The paid license is valid for one top-level domain and all its subdomains.
```

Comment:
```suggestion
Umbraco Plumber is a licensed product but does not require a purchase to use. New installs are defaulted to a trial license, while the paid license is available for purchase. The trial license introduces some restrictions around advanced features but is otherwise a full-featured workflow platform. The paid license is valid for one top-level domain and all its subdomains.
```